Sources/Apodini/Response/Response.swift

```diff
@@ -11,37 +11,64 @@ import ApodiniUtils
 
 public struct Response<Content: Encodable>: ResponseTransformable {
     public static var nothing: Response<Content> {
-        Response<Content>(connectionEffect: .open)
+        Response<Content>(information: InformationSet(), connectionEffect: .open)
     }
     
     public static var end: Response<Content> {
-        Response<Content>(connectionEffect: .close)
+        Response<Content>(information: InformationSet(), connectionEffect: .close)
```

Comment:
Could you please also add functions for initializing `end` with associated `Information` and `Status`?

Reply:
yeah, sure.